### DISCUSSION TITLE
Can this be used with Chadwm which has 7 patches already running?

### DISCUSSION THREAD

**Discussion by thorrrr** [bakkeby/dwm-flexipatch#368] [Q&A]
Hi 
Can this be used with Chadwm which has 7 patches already running? I am using Arcolinux Chadwn which is fork of DWM.
I only need 2 further patches and somebody pointed me here. But i think reading this is for more of a new install.
Or can it be used on a running DWM ?

**Comment by bakkeby** [comment]
This would be a separate install (or separate window manager if you want).

You can try manually patching Chadwm with the two patches you need (by manual patching I mean by hand, not using tools like `patch` or `git apply`, due to the likelihood of conflicts). 

If you want then you can also try out the patches using dwm-flexipatch before going down that rabbit hole. I'd recommend doing just a local `make` (rather than `sudo make install`) which leaves the `dwm` executable in the local directory, then running that through [Xephyr](https://wiki.archlinux.org/title/Xephyr).
